README.md

```diff
@@ -207,187 +250,226 @@ On first execution, this will generate golden snapshots. Subsequent runs will co
+- `reporter: 'dot' | 'line' | 'list'` - Choose a reporter: minimalist `dot`, concise `line` or detailed `list`. See [Folio reporters][folio-reporters] for more details.
+- `retries: number` - Each failing test will be retried up to the certain number of times.
+- `testDir: string` - Directory where test runner should search for test files.
+- `timeout: number` - Timeout in milliseconds for each test.
+- `workers: number` - The maximum number of worker processes to run in parallel.
 
-```diff
-// test/fixtures.ts
-import { folio as baseFolio } from "@playwright/test";
-import { LaunchOptions } from "playwright";
+### Skip tests with annotations
 
-const builder = baseFolio.extend();
+The Playwright test runner can annotate tests to skip under certain parameters. This is enabled by [Folio annotations][folio-annotations].
 
-builder.browserOptions.override(async ({ browserOptions }, runTest) => {
-  const modifiedOptions: LaunchOptions = {
-    ...browserOptions, // default
-    slowMo: 50,
-  }
-  await runTest(modifiedOptions);
+```ts
+test("should be skipped on firefox", async ({ page, browserName }) => {
+  test.skip(browserName === "firefox", "optional description for the skip");
+  // Test function
 });
+```
+
+### Run tests in parallel
 
-const folio = builder.build();
+Tests are run in parallel by default, using multiple worker processes. You can control the parallelism with the `workers` option in the configuration file or from the command line.
 
-+ export const it = folio.it;
-+ export const expect = folio.expect;
+```sh
+# Run just a single test at a time - no parallelization
+npx folio --workers=1
+
+# Run up to 10 tests in parallel
+npx folio --workers=10
 ```
 
 ```ts
-// test/index.spec.ts
-import { it, expect } from "./fixtures";
-
-// Test functions go here
-it("should have the slow mo", async ({ context }) => {
-  // ...
-});
+// config.ts
+import { PlaywrightTestConfig } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+  // No parallelization on CI, default value locally.
+  worker: process.env.CI ? 1 : undefined,
+  projects: [
+    // Your projects go here
+  ],
+};
+export default config;
 ```
 
-### Skip tests with annotations
+By default, test runner chooses the number of workers based on available CPUs.
 
-The Playwright test runner can annotate tests to skip under certain parameters. This is enabled by [Folio annotations][folio-annotations].
+### Reporters
 
-```js
-it("should be skipped on firefox", (test, { browserName }) => {
-  test.skip(browserName === "firefox", "optional description for the skip")
-}, async ({ page, browserName }) => {
-  // Test function
-});
-```
+Playwright test runner comes with a few built-in reporters for different needs and ability to provide custom reporters. The easiest way to try out built-in reporters is to pass `--reporter` [command line option](#command-line). Built-in terminal reporters are minimalist `dot`, concise `line` and detailed `list`.
```

Comment:
I think Playwright is quite unique that we have 1k+ tests. The majority of the users have maybe 20-30 tests. For them the list reporter might be the best, so I think having list by default would improve the usability so they see instant feedback.

Reply:
List is the default.